mapswipe_workers/tests/integration/test_transfer_results_invalid_results.py

```diff
@@ -0,0 +1,51 @@
+import json
+import os
+import unittest
+
+import set_up
+import tear_down
+
+from mapswipe_workers import auth
+from mapswipe_workers.firebase_to_postgres import transfer_results
+
+
+class TestTranserResultsProject(unittest.TestCase):
+    def setUp(self):
+        self.project_id = set_up.create_test_project(
+            "tile_map_service_grid", "build_area", results=False
+        )
+
+    def tearDown(self):
+        tear_down.delete_test_data(self.project_id)
+
+    def test_invalid_task_in_result(self):
```

Comment:
@laurentS here I test for an psycopg2 related error during the transfer of the results